ionic_user_interface/src/components/gym-selector/GymSelector.vue

```diff
@@ -114,11 +112,11 @@ export default defineComponent({
     const viewMap = ref(false);
 
     onMounted(async () => {
-      countryNameList.value = [...Lookup.countries.sort()];
       if (userCountry.value !== null) {
         selectedCountryIso3.value = userCountry.value.iso3;
         const countryGymLocations = await getGymsByCountry(userCountry.value.iso3);
         gymLocationList.value = countryGymLocations;
+        selectedGym.value = '1.343014966025054, 103.77590653585952'; // Z-Vertigo's gym location
```

Comment:
Need to check if the country is singapore. If not I put china also show zvertigo when reloaded

Reply:
Best is just remember their previously selected gym too

Reply:
Done